app/messaging/process-message.js

```diff
@@ -20,6 +20,4 @@ const processMessage = async (message, receiver) => {
   }
 }
 
-module.exports = {
-  processMessage
-}
+module.exports = processMessage
```

Comment:
Functions being exported from their file should not be wrapped in `{}` this this should be done in `index.js` module files when exporting multiple functions from within.